lib/hijack/wrap_ddp_stringify.js

```diff
@@ -0,0 +1,33 @@
+wrapStringifyDDP = function() {
+  var originalStringifyDDP = DDPCommon.stringifyDDP;
+
+  DDPCommon.stringifyDDP = function(msg) {
+    var msgString = originalStringifyDDP(msg);
+    var msgSize = msgString.length;
+
+    var kadiraInfo = Kadira._getInfo(null, true);
+
+    if(kadiraInfo) {
+      if(kadiraInfo.trace.type === 'method') {
+        Kadira.models.methods.trackMsgSize(kadiraInfo.trace.name, msgSize);
+      } else if(kadiraInfo.trace.type === 'sub') {
+        Kadira.models.pubsub.trackMsgSize(kadiraInfo.trace.name, "initialSent", msgSize);
+      }
+
+      return msgString;
+    }
+
+    if(Kadira.env.currentSub) {
+      if(Kadira.env.currentSub.__kadiraInfo){
+        Kadira.models.pubsub.trackMsgSize(Kadira.env.currentSub._name, "initialSent", msgSize);
```

Comment:
What's the diff between above initialSent and this one?
Are they the same. If so, we could remove the above one.